src/test/scala/com/stratio/hermes/utils/HermesTest.scala

```diff
@@ -64,15 +67,94 @@ class HermesTest extends FlatSpec with Matchers {
   it should "raise an exception when it gets a firstName/lastName and firstNames/lastNames are empty in the locale" in {
     val hermes = Hermes("XX")
     //scalastyle:off
-    an [NoSuchElementException] should be thrownBy hermes.Name.firstName()
-    an [NoSuchElementException] should be thrownBy hermes.Name.lastName()
+    an[NoSuchElementException] should be thrownBy hermes.Name.firstName()
+    an[NoSuchElementException] should be thrownBy hermes.Name.lastName()
     //scalastyle:on
   }
 
   it should "raise an exception when it tries to load a locale that don't exist" in {
     //scalastyle:off
-    val thrown = the [IllegalStateException] thrownBy Hermes("XY").Name.firstName()
+    val thrown = the[IllegalStateException] thrownBy Hermes("XY").Name.firstName()
+    //scalastyle:on
+    thrown.getMessage should equal(s"Error loading locale: /locales/name/XY.json")
+  }
+
+  it should "generate a random integer of 0 digit give it 0" in {
+    val hermesNum = Hermes("")
+    hermesNum.Number.number(0) shouldBe 0
+
+  }
+
+  it should "generate a random integer when it passed the number of digit" in {
+    val hermesNum = Hermes("")
+    forAll { (n: Int) =>
+      numberOfDigitsFromANumber(hermesNum.Number.number(n)) == n
+    }
+  }
+
+  it should "generate a random integer when it passed the number of digit and the sign" in {
+    val hermesNum = Hermes("")
+    //scalastyle:off
+    forAll { (n: Int) =>
+      hermesNum.Number.number(n, Positive) > 0
+    }
+    forAll { (n: Int) =>
+      hermesNum.Number.number(n, Negative) < 0
+    }
+    val num: Int = hermesNum.Number.number(2, Positive)
+    num > 0 shouldBe true
```

Comment:
Does it make sense to check these unary conditions when you have check a lot of cases using scalacheck before? I'd just keep the forAll's in this test.